NEXT_CHANGELOG.md

```diff
@@ -5,6 +5,8 @@
 ### Breaking Changes
 * Do not set `run_as from run_as_user_name in DLT pipelines. This fixes an issue where the value for run_as was unintentionally cached in the terraform state. More details and the workaround are specified in the PR: ([#4886](https://github.com/databricks/terraform-provider-databricks/pull/4886))
 
+* Add a new resource `databricks_service_principal_federation_policy` ([#4635](https://github.com/databricks/terraform-provider-databricks/pull/4635))
```

Comment:
How about moving this to the next `New Features and Improvements` section? 
I guess this will not break anything.